### PR TITLE
Display cookie info at /<scrapbook-id>/cookies

### PR DESCRIPTION
**Pull request by gjvis**

### CODE REVIEW

**Comment by gjvis** [comment]
This was accidentally removed in v2

File: app/views/static_pages/cookies.html.erb [hunk 1]
```diff
@@ -37,8 +37,8 @@
 
      <ul class="list list-bullet">
        <li>the pages you visit</li>
-       <li>how long you spend on each page </li>
-       <li>how you got to the site </li>
+       <li>how long you spend on each page</li>
+       <li>how you got to the site</li>
```
Comment (nathancashmoredwp) [inline app/views/static_pages/cookies.html.erb]: ?

**Comment by nathancashmoredwp** [comment]
As discussed ... pull the cookies in the main routings.

File: app/views/static_pages/cookies.html.erb (hunk 1 above)
Comment (gjvis) [inline app/views/static_pages/cookies.html.erb]: I removed some invisible characters

**Comment by gjvis** [comment]
updated, ping @nathancashmoredwp

**Comment by nathancashmoredwp** [comment]
:bikini:

**Pull request merged**